src/main/java/com/google/sps/servlets/ItemFeedbackServlet.java

```diff
@@ -0,0 +1,78 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.sps.data.ItemFeedback;
+import com.google.sps.data.ItemFeedbackService;
+import com.google.sps.data.Json;
+import com.google.sps.data.LearningPath;
+import com.google.sps.data.LearningPathService;
+import com.google.sps.data.User;
+import com.google.sps.data.UserService;
+
+import java.io.IOException;
+
+
+/**
+ * Handle requests for a single learning path
+ */
+@WebServlet({"/learning-path/*"})
```

Comment:
I think we can merge this and the other servlet ("/path/*") once we agree on the common name. I think they're different by accident.